src/config/featuredContent/featuredContentMetadata.js

```diff
@@ -1,43 +1,145 @@
+export const contentTypes = [
+    {
+        content_type: "Data Definitions",
+        color: "violet vivid",
+        secondary_color: "violet-60v"
+    },
+    {
+        content_type: "My USAspending Search",
+        color: "red-warm",
+        secondary_color: "vivid 50-v"
+    },
+    {
+        content_type: "See 4 Yourself",
+        color: "orange vivid",
+        secondary_color: "orange 30-v"
+    },
+    {
+        content_type: "Recently Answered Questions",
+        color: "indigo cool vivid",
+        secondary_color: "indigo cool 80v"
+    },
+    {
+        content_type: "What's the Difference?",
+        color: "mint",
+        secondary_color: "mint-30"
+    }
+];
```

Comment:
i'm not sure if this contentTypes object is necessary anymore since the hex values are added below.